src/tbb/CMakeLists.txt

```diff
@@ -71,6 +71,15 @@ if (NOT ("${CMAKE_SYSTEM_PROCESSOR}" MATCHES "(armv7-a|aarch64|mips|arm64|riscv)
     target_compile_definitions(tbb PRIVATE __TBB_USE_ITT_NOTIFY)
 endif()
 
+if (TBB_VERIFY_DEPENDENCY_SIGNATURE)
+    if (MSVC)
+        set(TBB_COMPILER_MACRO_SWITCH /D)
+    else()
+        set(TBB_COMPILER_MACRO_SWITCH -D)
+    endif()
+    set(TBB_LIB_COMPILE_FLAGS ${TBB_LIB_COMPILE_FLAGS} ${TBB_COMPILER_MACRO_SWITCH}__TBB_VERIFY_DEPENDENCY_SIGNATURE)
```

Comment:
Can `target_compile_definitions` be used instead?

Reply:
Of course! Thanks! Implemented.